lib/src/asn1/liblte_mme.cc

```diff
@@ -3019,6 +3019,9 @@ LIBLTE_ERROR_ENUM liblte_mme_unpack_emergency_number_list_ie(uint8**
     emerg_num_list->N_emerg_nums = 0;
     while (length < sent_length) {
       idx                                               = emerg_num_list->N_emerg_nums;
+      if(idx >= 12) {
```

Comment:
no magic numbers, better use `LIBLTE_MME_EMERGENCY_NUMBER_LIST_MAX_SIZE` to check against, also check the code formatting and add a descriptive log/error message